generators/entity/templates/src/main/webapp/app/entities/_entity-management-dialog.html

```diff
@@ -8,7 +8,7 @@ <h4 class="modal-title" id="my<%= entityClass %>Label" translate="<%= keyPrefix
     </div>
     <div class="modal-body">
         <jhi-alert-error></jhi-alert-error>
-        <div class="form-group">
+        <div class="form-group" ng-show="showIdValue">
```

Comment:
couldn't this be just a simple `ng-show="vm.<%=entityInstance %>.id"` so that ID will be shown if it exists else will be hidden.